docs/ref/files/storage.txt

```diff
@@ -11,7 +11,18 @@ Django provides convenient ways to access the default storage class:
 
 .. data:: storages
 
-    Storage instances as defined by :setting:`STORAGES`.
+    A dictionary-like object that allows you to retrieve a storage instance
```

Comment:
I believe avoiding the use of pronouns like "you" makes the documentation more direct and objective, as it emphasizes the capabilities of a program/code rather than addressing the reader.
```suggestion
    A dictionary-like object that allows to retrieve a storage instance
```

Reply:
I'd either go `allows retrieving` or drop the allows altogether and just have `to retrieve`. As you think.